Makefile

```diff
@@ -0,0 +1,66 @@
+-include config.mk
```

Comment:
Makefiles such as this one (I use very similar ones for most of my packages) are the reason why my packages have significantly fewer compile warnings than other elisp. It's just very simple to type `make` and learn whether you only didn't get any warnings when compiling from within Emacs because you happened to also have the unspecified dependencies loaded among other repl vs. separate compile step issues.

Yes, it is a a simple Makefile, but that doesn't mean it is worthless. Rather that's the result of reducing it to the essential parts over the course of a few wears. Other people use Cask instead. That looks much more impressive not least because it adds a dependency on Python and comes with a manual and does project specific dependency management (which makes sense for the build systems that idea was borrowed from, much less for Emacs which has one global scope).

Reply:
Yup, cool. I'd be happy to end up with a Travis build for this, and the makefile would be useful in that case.

Reply:
I've started working on that but need you to enable it at https://travis-ci.org/profile/melpa.